utils.sh

```diff
@@ -88,6 +88,11 @@ function create_cluster() {
       cp -rf ${ASSETS_EXTRA_FOLDER}/*.yaml ${assets_dir}/openshift/
     fi
 
+    if [[ "$BMO_WATCH_ALL_NAMESPACES" == "true" ]];
+    then
```

Comment:
nit: then should be on the precedent line to be consistent with other if-then blocks